src/main/java/utils/errorhandling/ExceptionHandling.java

```diff
@@ -0,0 +1,9 @@
+package utils.errorhandling;
+
+public class ExceptionHandling {
+    public static void IllegalArgumentException(int exitCode) {
+        System.out.println("Illegal argument exception");
```

Comment:
Why are you doing this instead of throwing exception?

Reply:
This is fixed.